llvm/lib/Target/RISCV/RISCVInstrInfoZvfbf.td

```diff
@@ -17,15 +17,15 @@
 // Instructions
 //===----------------------------------------------------------------------===//
 
-let Predicates = [HasStdExtZvfbfmin], Constraints = "@earlyclobber $vd",
+let Predicates = [HasStdExtZvfbfminOrZvfbfa], Constraints = "@earlyclobber $vd",
```

Comment:
This instruction is not part of Zvfbfa. It is only mentioned in the spec to say that `vfwcvtbf16.f.f.v` with altfmt has the same behavior as `vfwcvtbf16.f.f.v`.